packages/core/src/Room.ts

```diff
@@ -53,19 +60,59 @@ export abstract class Room<State= any, Metadata= any> {
   }
 
   public listing: RoomListingData<Metadata>;
+
+  /**
+   * A ClockTimer instance, used for timing events.
+   */
   public clock: Clock = new Clock();
 
+  /**
+   * A unique, auto-generated, 9-character-long id of the room.
+   * You may replace `this.roomId` during `onCreate()`.
+   */
   #_roomId: string;
+  /**
+   * The name of the room you provided as first argument for `gameServer.define()`.
+   */
   #_roomName: string;
```

Comment:
Only "userland" properties should be documented, we can move these to `roomId` and `roomName` getters. Private properties (starting with `#`) are not exposed, even when using `extends`